pkg/api/pod/util_test.go

```diff
@@ -4136,7 +4136,7 @@ func TestValidateAllowPodLifecycleSleepActionZeroValue(t *testing.T) {
 					},
 				},
 			},
-			expectAllowPodLifecycleSleepActionZeroValue: false,
+			expectAllowPodLifecycleSleepActionZeroValue: true,
```

Comment:
This test should probably run each case with the gate set to on and off and assert the expected result.  As it is, there's no longer any coverage for the gate being disbled?

Reply:
Ah sorry I missed this. This unit test was added recently in a bugfix PR and I realized that the feature gate logic was not added at all. I've done this in https://github.com/kubernetes/kubernetes/pull/130621/commits/a723ce30e638c5fce2309f4170179330d52a45d9. All the tests run with both the the feature gate turned on and off now.